extensions/git/src/commands.ts

```diff
@@ -1717,19 +1717,27 @@ export class CommandCenter {
 
 	@command('git.commitMessageAccept')
 	async commitMessageAccept(arg?: Uri): Promise<void> {
-		if (!arg) { return; }
```

Comment:
How about something like this:
```TS
if (!arg && !window.activeTextEditor) { return; }
arg ??= window.activeTextEditor!.document.uri;
```

Reply:
Thanks, your version is definitely better